### PR TITLE
a new model 

### PR DESCRIPTION
**Pull request by philschmid**

### CODE REVIEW

**Review by alvarobartt — APPROVED** [review]
LGTM!

File: llama31.md [hunk 1]
```diff
@@ -0,0 +1,946 @@
...
+Among the features and integrations being released, we have:
+
+
+* [Models on the Hub](https://huggingface.co/collections/meta-llama/llama-31-release-669fc079a0c406a149a5738f)
+* Hugging Face Transformers and TGI integration
+* [Hugging Chat integration for Meta Llama 3.1 405B Instruct](https://huggingface.co/chat/models/meta-llama/Meta-Llama-3.1-405b-instruct/)
+* Inference & Deployment Integration with Inference Endpoints, Google Cloud, Amazon SageMaker & DELL Enterprise Hub
+* Quantization for FP8, AWQ and GPTQ for easier inference
+* Fine-tuning Llama 3.1 8B on a single GPU with 🤗 TRL
+* Generate synthetic data using Llama 3.1 70B and 405B with Distilabel
```
Comment (pcuenca) [inline llama31.md:33]: Maybe mention the recipes here as well? (unsure)

File: llama31.md [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+* [Meta-Llama-3.1-405B-Instruct](https://huggingface.co/meta-llama/Meta-Llama-3.1-405B-Instruct): Instruct fine-tuned version of the base 405B model
+
+In addition to these 6 language models, Llama Guard 3 and Prompt Guard were released. 
+* [Llama Guard 3](https://huggingface.co/meta-llama/Llama-Guard-3-8B) is the latest iteration in the Llama Guard family, fine-tuned on Llama 3.1 8B. It is built for production use cases, with a 128k context length and multilingual capabilities. Llama Guard 3 can classify LLM inputs (prompts) and responses to detect content that would be considered unsafe in a risk taxonomy. 
+* [Prompt Guard](https://huggingface.co/meta-llama/Prompt-Guard-86M), on the other hand, is a small 279M parameter BERT-based classifier that can detect prompt injection and jailbreaking. It was trained on a large corpus of attacks and is suggested to be further fine-tuned with application-specific data.
+
+New in Llama 3.1 compared to Llama 3 is that the instruct models are fine-tuned on tool calling for agentic use cases. There are two built-in tools (search, mathematical reasoning with Wolfram Alpha) that and can be expanded with custom JSON functions.
```
Comment (pcuenca) [inline llama31.md:82]:
```suggestion
New in Llama 3.1 compared to Llama 3 is that the instruct models are fine-tuned on tool calling for agentic use cases. There are two built-in tools (search, mathematical reasoning with Wolfram Alpha) that can be expanded with custom JSON functions.
```

File: _blog.yml [hunk 3]
```diff
@@ -4427,4 +4427,15 @@
     - guide
     - llm
     - swift
-    - wwdc
\ No newline at end of file
+    - wwdc
+
+- local: llama31
```
[the remark this answers is not in the document]
Comment (philschmid) [reply _blog.yml:4432]: cannot update is already shared with partners and similar to previous blogs.

**Pull request merged**